mem_storage/src/main/java/com/splicemachine/storage/MPartition.java

```diff
@@ -502,9 +502,6 @@ private void delete(MDelete mDelete,Lock rowLock) throws IOException{
 
     private NavigableSet<DataCell> getAscendingScanSet(DataScan scan){
         NavigableSet<DataCell> dataCells;
-        if(memstore.size()<=0)
```

Comment:
Would it make sense to substitute it for isEmpty() ? It might not be constant-time either but it would be much faster than .size()